neon_utils/mycroft_override_functions.py

```diff
@@ -0,0 +1,148 @@
+import time
+
+from mycroft_bus_client import Message
+
+from neon_utils.logger import LOG
+
+
+def speak(utterance, expect_response=False, message=None, private=False, speaker=None, wait=False, meta=None):
```

Comment:
this probably belongs in skill_override_functions since it is part of the skill class

Reply:
my reasoning was that speak is a method in MycroftSkill. moving it to skill_override_functions makes sense though, since there are several skill that override this method

Reply:
My idea with skill_override_functions was that it could include all the `MycroftSkill` methods we need to add or override.

Actually, it looks like I did add this since you created your fork; you may want to rebase your branch to see any other changes.

https://github.com/NeonGeckoCom/neon-skill-utils/blob/5d7d0f61ac595f2b94b773b59cc2460749f14fb3/neon_utils/skill_override_functions.py#L123